conda_forge_tick/auto_tick.py

```diff
@@ -401,7 +403,12 @@ def add_arch_migrate(migrators: MutableSequence[Migrator], gx: nx.DiGraph) -> No
             graph=total_graph,
             pr_limit=PR_LIMIT,
             name="arm osx addition",
-            piggy_back_migrations=[UpdateConfigSubGuessMigrator()],
+            piggy_back_migrations=[
```

Comment:
@CJ-Wright, is this not enough?

Reply:
The class currently doesn't take that as an arg.